Automattic/themes
# Course Theme - Fix post title spacings for course and lesson default

### Changes proposed in this Pull Request:
- Fixed the spacings above and below the post title of lessons and courses for course theme to match the design.
- We've changed only spacings specific to the default templates of Course and Lesson, because if any user already changed to another template choosing that style, we don't want to break their existing look. The default look will not be very different either.

### Testing instructions:
- Checkout this branch https://github.com/Automattic/sensei/pull/7063
- Create a course with a lesson
- Make sure learning mode is disabled for that course
- Now take the course and the lesson as a student
- Make sure the spacings above and below the Post Title matches the design

The top spacing is calculated as the spacing between the bottom of the site title to the top of the post title.
#### Design

Mobile:
(Top)
![Screenshot 2023-08-07 at 5 59 06 PM](https://github.com/Automattic/themes/assets/6820724/c22eee5c-60d2-46b8-b99a-4fa783e5d467)

(Bottom)
![Screenshot 2023-08-07 at 9 46 16 PM](https://github.com/Automattic/themes/assets/6820724/32bfe325-f2f8-4f32-b3c6-5ed8f2bc3069)

Desktop:
(Top)
![Screenshot 2023-08-07 at 9 50 32 PM](https://github.com/Automattic/themes/assets/6820724/51eb28b3-91d3-42e4-a73c-db8747b09b48)
(Bottom)
![Screenshot 2023-08-07 at 9 50 44 PM](https://github.com/Automattic/themes/assets/6820724/54f68fa8-006c-4c0d-9183-e3cad422021c)

Course Desktop:
![Screenshot 2023-08-07 at 5 53 20 PM](https://github.com/Automattic/themes/assets/6820724/d0db7693-d5e3-4f03-a80e-800fd502aab9)
![Screenshot 2023-08-07 at 5 53 37 PM](https://github.com/Automattic/themes/assets/6820724/dafe61b4-cc6a-47cb-a1ce-464d365b8c5d)

Lesson Desktop:
![Screenshot 2023-08-07 at 5 56 52 PM](https://github.com/Automattic/themes/assets/6820724/434d8da3-8b21-4f1f-bee0-8e3707f8b596)
![Screenshot 2023-08-07 at 5 57 03 PM](https://github.com/Automattic/themes/assets/6820724/b0e22df7-9ecf-41e5-aa99-bac32b3819c8)

Course Mobile:
![Screenshot 2023-08-07 at 5 54 38 PM](https://github.com/Automattic/themes/assets/6820724/a8213c72-0fbd-497d-964a-ad125fc4f5e7)
![Screenshot 2023-08-07 at 5 57 52 PM](https://github.com/Automattic/themes/assets/6820724/91dba65b-1ed3-4130-bf2b-d7d7c11879e0)

Lesson Mobile:
![Screenshot 2023-08-07 at 5 57 37 PM](https://github.com/Automattic/themes/assets/6820724/e230cf4f-2e23-45d9-9509-d70f288095a5)
![Screenshot 2023-08-07 at 5 57 52 PM](https://github.com/Automattic/themes/assets/6820724/907e144d-e195-4dcd-a0db-62440abe9c3e)

#### Related issue(s):

https://github.com/Automattic/themes/issues/7144
